va/va.go

```diff
@@ -113,12 +115,26 @@ func initMetrics(stats metrics.Scope) *vaMetrics {
 		[]string{"oid"},
 	)
 	stats.MustRegister(tlsALPNOIDCounter)
+	http01Fallbacks := prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "http01_fallbacks",
+			Help: "Number of IPv6 to IPv4 HTTP-01 fallback requests made",
+		})
+	stats.MustRegister(http01Fallbacks)
+	http01Redirects := prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "http01_redirects",
```

Comment:
I included this new stat because there are always rumblings in the CABF about redirects and I think it would be beneficial to have a sense of how many we're following.